### ISSUE TITLE
Adopt Function

### ISSUE THREAD

**Issue by nelsencd** [malle-pietje/UniFi-API-browser#18]
I have created an adoption function that you could add to you API.

 ``` public function adopt_ap($site_id, $mac) {
     if (!$this->is_loggedin) return FALSE;
     $mac             = strtolower($mac);
     $return          = FALSE;
     $json            = json_encode(array('mac' => $mac, 'cmd' => 'adopt'));
     $content_decoded = json_decode($this->exec_curl($this->baseurl.'/api/s/'.$this->site.'/cmd/devmgr','json='.$json));
     if (isset($content_decoded->meta->rc)) {
        if ($content_decoded->meta->rc == 'ok') {
           $return = TRUE;
        }
     }
       return $return;
  }
```

**Comment by malle-pietje** [comment]
Thanks for this!

I'll add it to the next release but I'm thinking of dropping $site_id as a parameter. Since we always are operating within the scope of a specific site within the class instance, adoption of an AP to the current site ($this->site) make most sense. Also, you don't appear to be using it within the function/method.
Do you agree?

**Comment by nelsencd** [comment]
I have another php script that I use to call this function to run it. That's also why I have the $site_id parameter.

**Comment by malle-pietje** [comment]
Ah, I see, would not be necessary in the public version then. Thanks again for the contribution!

**Comment by malle-pietje** [comment]
published with API client class version 1.0.13
